comp/metadata/clusteragent/impl/cluster_agent.go

```diff
@@ -211,6 +212,13 @@ func (dca *datadogclusteragent) getMetadata() map[string]interface{} {
 			dca.metadata["is_leader"] = leaderEngine.IsLeader()
 		}
 	}
+
+	// Add cluster check runner count
+	// Import "github.com/DataDog/datadog-agent/pkg/clusteragent/clusterchecks"
+	if count, err := clusterchecks.GetCLCRunnerCount(); err == nil {
+		dca.metadata["cluster_check_runner_count"] = count
```

Comment:
Is this enough, or do we also need the count of node agents running cluster checks?
I’m thinking that if this count is 0, we won’t be able to distinguish between a setup that’s running the cluster checks on the DCA and one that’s running them on the node agents.